haiku-games/gloom3d/gloom3d-1.0.0.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="Port of the Boom3D Quake-like engine to Haiku"
+DESCRIPTION="Gloom3D is the port Boom3D engine for Haiku. \
+It uses DirectWindow for quick draw window."
+HOMEPAGE="https://github.com/HaikuArchives/Gloom3D"
+COPYRIGHT="2000 Sasha Chukov"
+LICENSE=""
```

Comment:
You need to set a license here

Reply:
@Begasus Unfortunately, I couldn't find one so I have already wrote in title that I need license for it.

Reply:
Granted @arnavbhatt288 this one isn't your fault :+1: